src/partials/image.mjml

```diff
@@ -5,3 +5,25 @@
 {% macro test(text='nothing') %}
   <mj-text>{{ text }}</mj-text>
 {% endmacro %}
+
+{% macro grid(
```

Comment:
I think these should be handled with rows/columns rather than an explicit macro.